web/modules/custom/dpl_update/dpl_update.install

```diff
@@ -399,3 +399,10 @@ function dpl_update_update_10035(): string {
 function dpl_update_update_10036(): string {
   return _dpl_update_install_modules(['state_log']);
 }
+
+/**
+ * Adding autowire_plugin_trait for BNF.
+ */
+function dpl_update_update_10037(): string {
+  return _dpl_update_install_modules(['autowire_plugin_trait']);
```

Comment:
Is it a problem that this module is enabled before we go live with BNF?

Reply:
No, it only provides a PHP trait. Haven't brought it up with @arnested, but it could just have been a composer package (thought that wouldn't provide it any visibility on drupal.org).